api/lql_test.go

```diff
@@ -151,10 +171,26 @@ func TestQueryCreateOK(t *testing.T) {
 	_ = json.Unmarshal([]byte(mockResponse), &createExpected)
 
 	var createActual api.QueryResponse
-	createActual, err = c.V2.Query.Create(newQuery)
+	createActual, err = c.V2.Query.Create(testQuery)
 	assert.Nil(t, err)
 
 	assert.Equal(t, createExpected, createActual)
+
+	if strings.Contains(expectedResponseData, "queryLanguage") {
```

Comment:
If we are expecting a queryLanguage in the response, should we be validating that it's the right one?